src/person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def can_use_services?
+    return true if of_age? || @parent_permission
+
+    false
+  end
+
+  private
+
+  def of_age?
+    return true if @age >= 18
+
+    false
```

Comment:
- [ ] [OPTIONAL] No need to write the true or false, You can use it directly 
```
def of_age?
    @age >= 18
  end
```

Reply:
thank you!